### PR TITLE
tests/lib/cla_check: default to Python 3, tweaks, formatting

### PR DESCRIPTION
**Pull request by bboozzoo**
Use Python 3 by default. Apply little tweaks. Reformat with black.

### CODE REVIEW

**Review by zyga — APPROVED** [review]
Looks good

**Comment by bboozzoo** [comment]
@cmatsuoka can you install python3-launchpadlib in the workers that you host?

File: tests/lib/cla_check.py [hunk 1]
```diff
@@ -14,9 +14,11 @@
 try:
     from launchpadlib.launchpad import Launchpad
 except ImportError:
-    sys.exit("Install launchpadlib: sudo apt install python-launchpadlib python3-launchpadlib")
+    sys.exit(
+        "Install launchpadlib: sudo apt install python-launchpadlib python3-launchpadlib"
```
Comment (cmatsuoka) [inline tests/lib/cla_check.py:18]: Do we still need python-launchpadlib now that we switched to python3?

**Review by cmatsuoka — APPROVED** [review]
Thanks Maciek, left a question about python-launchpadlib.

**Comment by cmatsuoka** [comment]
> @cmatsuoka can you install python3-launchpadlib in the workers that you host?

It's there already.

**Review by anonymouse64 — APPROVED** [review]
lgtm, thanks for this

**Pull request merged**